java/Oleksii-Riabov/src/com/zoolatech/lecture7/tasks/_2/AtomicCalculator.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture7.tasks._2;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class AtomicCalculator {
+
+    private AtomicInteger value;
+
+    public AtomicCalculator(int value) {
+          this.value = new AtomicInteger(value);
+    }
+
+    public synchronized void add(int value2) {
```

Comment:
since you work with atomic objects, you don't need to use explicit locks for your methods

Reply:
"synchronize" modificators already removed.